vagga.yaml

```diff
@@ -51,6 +51,18 @@ containers:
       /run: !Tmpfs
         subdirs:
           postgresql: { mode: 0o777 }
+  test:
+    setup:
+      - !Ubuntu xenial
```

Comment:
I'm gonna dig into it myself.
![](http://www.quickmeme.com/img/e7/e7633bedf897bb24ce668ac9c5df6bf88a58ff7e114d27606a756f4c4888a3f1.jpg)